client/dashboard/sites/hosting-feature-activation-modal/error-content-info.tsx

```diff
@@ -0,0 +1,206 @@
+import {
+	__experimentalVStack as VStack,
+	Card,
+	CardBody,
+	CardDivider,
+	CardHeader,
+	ExternalLink,
+} from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { Fragment } from 'react';
+import { Notice } from '../../components/notice';
+import { Text } from '../../components/text';
+import type { NoticeVariant } from '../../components/notice/types';
+import type { AutomatedTransferEligibilityError } from '@automattic/api-core';
+
+enum EligibilityErrors {
```

Comment:
I think we should stop using enum and start using const instead.

Reply:
Updated to 

```const EligibilityErrors = {
	BLOCKED_ATOMIC_TRANSFER: 'blocked_atomic_transfer',
	TRANSFER_ALREADY_EXISTS: 'transfer_already_exists',
	NO_BUSINESS_PLAN: 'no_business_plan',
	NO_JETPACK_SITES: 'no_jetpack_sites',
	NO_VIP_SITES: 'no_vip_sites',
	SITE_GRAYLISTED: 'site_graylisted',
	NON_ADMIN_USER: 'non_admin_user',
	NOT_RESOLVING_TO_WPCOM: 'not_resolving_to_wpcom',
	NO_SSL_CERTIFICATE: 'no_ssl_certificate',
	EMAIL_UNVERIFIED: 'email_unverified',
	EXCESSIVE_DISK_SPACE: 'excessive_disk_space',
	IS_STAGING_SITE: 'is_staging_site',
} as const;

type EligibilityErrors = ( typeof EligibilityErrors )[ keyof typeof EligibilityErrors ];```

Let me know if this is what you had in mind.

Reply:
Yes, thanks!